tools/tools.go

```diff
@@ -0,0 +1,9 @@
+// +build tools
+
+// tools is a dummy package that will be ignored for builds, but included for dependencies
+package tools
+
+import (
+	_ "k8s.io/code-generator/cmd/deepcopy-gen"
+	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"
```

Comment:
This is just for my info? Is this a new tool that must be used @luisdavim

Reply:
I am trying to figure out the use of controller-gen. Is this internally used by update-codegen.sh?

Reply:
Don't we use it to generate the CRDs from the go structs?

Reply:
We don't currently. The ones in manifests folder are manual.

Reply:
This will be a good one to add as a make target against the manifests folder. Or perhaps a separate PR if this PR has taken lot of your time.

Reply:
ok, I'll look into that.

Reply:
I've updated the makefile to also generate the CRD manifests